integrations/aws-v3/pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "aws-v3"
-version = "0.3.4-dev"
+version = "0.3.4"
```

Hey👋 
Wont this bump to GA.
```suggestion
version = "0.3.4-beta"
```